### DISCUSSION TITLE
Is the chocolatey package kept up to date?

### DISCUSSION THREAD

**Discussion by Luxvao** [podman-desktop/podman-desktop#714] [Q&A] [answered]
I searched for podman-desktop on chocolatey.org and I found it says that the version is 0.0.6 is the choco package kept up to date?

**Comment by benoitf** [comment, marked as answer]
@Luxvao 
Thanks for the feedback.
There is validation on chocolatey.org so it's taking a huge to make updates for now

https://community.chocolatey.org/packages/podman-desktop
I submitted 0.6.0 at the beginning of september and it has been validated only a couple of weeks ago
![image](https://user-images.githubusercontent.com/436777/198274819-634bae18-fb9d-4cc7-9400-3190aa72c988.png)

During that time it was not possible to update a new package.

I hope that we'll get the `trusted package` flag so we could update in a timely manner.

**Comment by benoitf** [comment]
On Winget and Scoop the packages are reviewed/updated in 1-2 days.

**Comment by benoitf** [comment]
For 0.9.1 it's in pending state https://community.chocolatey.org/packages/podman-desktop/0.9.1
